CHANGELOG.md

```diff
@@ -14,6 +14,11 @@
 - Add long-distance restrictions via `<<` and `>>` operators. #943
 - Fix relative pathing for `mkdocstring-python=>1.9.1`. #967, #968
 
+### Pipelines
+
+- DLC
+   - Allow dlc without pre-existing tracking data #950
```

Comment:
Other lines reflect PR number out of my assumption that a user would want to see the set of changes from the PR, and can follow up on the linked issues for the motivation

```suggestion
   - Allow dlc without pre-existing tracking data #970
```